strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/test/java/org/carlspring/strongbox/artifact/coordinates/PypiWheelArtifactCoordinatesTest.java

```diff
@@ -0,0 +1,267 @@
+package org.carlspring.strongbox.artifact.coordinates;
+
+import org.carlspring.strongbox.artifact.coordinates.PypiWheelArtifactCoordinates;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+import java.util.*; 
+
+/**
+ * Class to test the functionality of PypiArtifactCoordinates and PypiArtifactCoordinatesUtils
```

Comment:
As far as I can see there is no longer a `PypiArtifactCoordinates`, so this should be renamed as well.
Could you please double-check the same for the `PypiArtifactCoordinatesUtils`?

Thanks! :)